src/dotnet/Chat.UI.Blazor/Components/ChatListNavbarWidget.razor

```diff
@@ -9,62 +9,116 @@
         + (mustPlayPinned ? "" : " opacity-50")
         + (isPlaying ? " on" : mustPlayPinned ? "" : " off");
 
-    RenderFragment RenderItem(Chat c) =>
+    RenderFragment RenderUnpinned(Chat c) =>
```

Comment:
Component is quite big. May be it make sense to split it in several components?

Reply:
Agree, makes sense.